Sources/Xcodeproj/generate().swift

```diff
@@ -70,13 +70,27 @@ public func generate(dstdir: String, projectName: String, srcroot: String, modul
     return xcodeprojPath
 }
 
-private func open(_ path: String..., body: ((String) -> Void) throws -> Void) throws {
+import class Foundation.NSData
+
+func open(_ path: String..., body: ((String) -> Void) throws -> Void) throws {
```

Comment:
This function should get a comment explaining the behavior that it doesn't overwrite unless the contents have changed.